pkg/kubelet/kubelet.go

```diff
@@ -2197,6 +2197,15 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 // can be admitted, a brief single-word reason and a message explaining why
 // the pod cannot be admitted.
 func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, string) {
+	if rs := kl.runtimeState.networkErrors(); len(rs) != 0 {
+		hostNetwork := false
+		if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.HostNetwork {
```

Comment:
how about instead:
```
if (!podUsesHostNetwork(pod)) {
    return false, "NetworkNotReady", fmt.Sprintf(...)
}
```
We added podUsesHostNetwork() specifically to cut down on this repeated pattern :)